### PR TITLE
Updated Contributing.md

### PR DESCRIPTION
**Pull request by N3tt3rm4n**
I added some code style preferences ;) Please change them if you don't like them :sunglasses: 

Btw I will have a closer look at the translations in core in the following days 'cause I don't have much time today

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -31,6 +31,10 @@ Here, we see that "global" is the "category", it is used in the code to identify
 
 "Encrypt your files" is the translation text, it's what you need to translate. (You can overwrite)
 
+### Code Style
+
+Please use following style: Intend mode `Spaces` and Intend size `4`. If you have done this (you can adjust that at the top of the file right of `Edit file`) you can use tabs, they automatically will turn into four spaces.
```
Comment (Super-Baleine) [inline CONTRIBUTING.md]: For the GitHub's web editor. You've to precise x)

**Comment by Super-Baleine** [comment]
okay x)

**Review by Super-Baleine — CHANGES_REQUESTED** [review]
Precise "indent mode 4 ..." for the GitHub's web code editor

**Comment by N3tt3rm4n** [comment]
Like that?

**Review by Super-Baleine — APPROVED** [review]
Good.

**Comment by Super-Baleine** [comment]
Thanks man!

**Pull request merged**